src/javaRosa.js

```diff
@@ -1723,6 +1750,19 @@ define([
             // will be properly set as such.
             // 4. duplicate itext ids will be automatically updated to create
             // non-duplicates
+            
+            function hashtags(outputRef) {
+                var value = $(outputRef).attr('value') || $(outputRef).attr('ref'),
+                    key = !!$(outputRef).attr('value') ? 'value' : 'ref',
+                    parsed = xpath.parser.parse(value),
+                    hashtag = parsed.toHashtag(),
+                    xpath_ = parsed.toXPath();
+                if (xpath_ === hashtag) {
+                    return '<output ' + key + '="' + xpath_ + '"/>';
+                } else {
+                    return '<output ' + key + '="' + xpath_ + '" vellum:' + key + '="' + hashtag + '"/>';
```

Comment:
Is it possible that `xpath_` could ever contain a quote character (`"`)? If it does this will construct a broken tag.